v3 address-validation release (#138)
department-of-veterans-affairs/developer-portal-content: content/verification/address-validation/release-notes/2024-05-15.md

> Released version 3 of address validation. Updates include:

Reviewer: ```suggestion
We released version 3 (v3) of address validation. Updates include:
```

- `/validate` and `/candidate` endpoints

>      - Removed support for US Congressional District. 

Reviewer: Should this be a sub-bullet  of 'validate' and 'candidate above? Or its own bullet point?
```suggestion
     - Removing support for US Congressional District. 
```

>      - Added country code support for both International Standards Organization (ISO)  and Federal Information Processing (FIPS) standards.

Reviewer: Should this be a sub-bullet of 'validate' and 'candidate above? Or its own bullet point?

```suggestion
     - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing (FIPS) standards.
```

Author: yes, these are sub-bullets describing changes to the 2 endpoints.

> - `/cityStateProvince` endpoint is renamed to `/locality` to reflect the capability.

Reviewer: Reflect the capability of what?

Author: reflects the capability of the endpoint.

> - For ease of mapping,  fields are renamed to be identical between this service and contact information service version 2.

Reviewer: ```suggestion
- For ease of mapping,  fields are renamed to be identical between this service and contact information service in version 2 of the API.
```

> We released version 3 (v3) of address validation. Updates include:

Reviewer: ```suggestion
We released version 3 (v3) of the Address Validation API. Updates include:
```

> - `/validate` and `/candidate` endpoints

Reviewer: ```suggestion
- Adding `/validate` and `/candidate` endpoints.
```

Author: these endpoints existed in v1 of their API, so I don't think we can say they are being added here.  v3 basically takes functionality of v1 and v2 and combines them into v3 and describes the major changes such as congressional district and countrycode ISO style that is now different in v3 vs v1

Reviewer: Got it, I can remove this comment.

> - `/validate` and `/candidate` endpoints

Reviewer: ```suggestion
- `/validate` and `/candidate` endpoints.
```

>      - Removing support for US Congressional District. 

Reviewer: @dsteeve are these two (Removing support...) and (Adding country code..) supposed to be sub-bullets of '/validate' and '/candidate'? Just want to double check.

Author: They are sub-bullets to these endpoints.  If you look at address-validation v1, there was a congressional district param, and that is no longer there, and country code handling is now different as well.

Reviewer: Awesome, I'll resolve!

>      - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing (FIPS) standards.

Reviewer: ```suggestion
     - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing Standards (FIPS). 
```

Reviewer: @dsteeve I believe this is the Federal Information Processing Standards, so it should all be capitalized?

> - `/cityStateProvince` endpoint is renamed to `/locality` to reflect the capability of the endpoint.

Reviewer: ```suggestion
- Renaming `/cityStateProvince` endpoint to `/locality` to reflect the capability of the endpoint.
```

> - For ease of mapping,  fields are renamed to be identical between this API and Contact Information API version 2 (v2).

Reviewer: ```suggestion
- For ease of mapping, fields are renamed to be identical between this API and Contact Information API version 2 (v2).
```
